### PR TITLE
Operations opt split

### PR DESCRIPTION
**Pull request by vgorkavenko**

### CODE REVIEW

File: ethereum-validators-set/src/subagents/exitbus-oracle/agent-exitbus-oracle.ts [hunk 1]
```diff
@@ -182,18 +183,7 @@ async function getReportProcessingStarted(blockFrom: number, blockTo: number) {
...
-    ORACLE_REPORT_SANITY_CHECKER_ABI,
-    ethersProvider,
-  );
-
-  const { maxValidatorExitRequestsPerReport } =
-    await sanityChecker.getOracleReportLimits({ blockTag: block });
-
-  lastMaxValidatorExitRequestsPerReport = maxValidatorExitRequestsPerReport;
-}
+async function updateMaxValidatorExitRequestsPerReport(block: number) {}
```
Comment (madlabman) [inline ethereum-validators-set/src/subagents/exitbus-oracle/agent-exitbus-oracle.ts:186]: Abandoned?

File: ethereum-validators-set/src/subagents/node-operators-registry/agent-node-operators-registry.ts [hunk 2]
```diff
@@ -314,11 +309,12 @@ function handleStakeLimitSet(txEvent: TransactionEvent, findings: Finding[]) {
 export async function handleBlock(blockEvent: BlockEvent) {
   const findings: Finding[] = [];
 
-  await handleStuckPenaltyEnd(blockEvent, findings);
-
-  if (blockEvent.blockNumber % 100) {
+  if (blockEvent.blockNumber % BLOCK_INTERVAL == 0) {
```
Comment (madlabman) [inline ethereum-validators-set/src/subagents/node-operators-registry/agent-node-operators-registry.ts:312]: 🫠

File: ethereum-validators-set/src/subagents/sanity-checker/agent-sanity-checker.ts [hunk 3]
```diff
@@ -38,44 +37,54 @@ export async function initialize(
...
+      eventName.includes(event.name),
+    );
+    if (!eventSign) return;
+    const limitName = (ORACLE_REPORT_SANITY_CHECKER_LIMITS_EVENTS as any)[
+      eventSign
+    ];
+    if (
+      limitName &&
+      new BigNumber(String(prevLimits[limitName])) !=
+        new BigNumber(String(event.args[limitName]))
```
Comment (madlabman) [inline ethereum-validators-set/src/subagents/sanity-checker/agent-sanity-checker.ts:84]:
```
> new BigNumber(1) != new BigNumber(0)
true
> new BigNumber(1) != new BigNumber(1)
true
>
```

Strings comparison should be enough I think

Comment (vgorkavenko) [reply ethereum-validators-set/src/subagents/sanity-checker/agent-sanity-checker.ts]: fixed

File: ethereum-validators-set/src/subagents/exitbus-oracle/agent-exitbus-oracle.ts (hunk 1 above)
Comment (vgorkavenko) [reply ethereum-validators-set/src/subagents/exitbus-oracle/agent-exitbus-oracle.ts]: fixed

**Review by madlabman — APPROVED** [review]

File: ethereum-governance/package.json
Comment (dgusakov) [inline ethereum-governance/package.json:1]: Package naming

File: ethereum-steth/src/subagents/steth-ops/agent-steth-ops.ts [hunk 4]
```diff
@@ -105,59 +89,14 @@ export async function handleBlock(blockEvent: BlockEvent) {
   const findings: Finding[] = [];
 
   await Promise.all([
-    handleNodeOperatorsKeys(blockEvent, findings),
     handleBufferedEth(blockEvent, findings),
     handleDepositExecutorBalance(blockEvent, findings),
-    handleStakingLimit(blockEvent, findings),
-    handleMevRelayCount(blockEvent, findings),
-    handleEnsNamesExpiration(blockEvent, findings),
+    handleStakingLimit(blockEvent, findings), //
```
Comment (dgusakov) [inline ethereum-steth/src/subagents/steth-ops/agent-steth-ops.ts:94]:
```suggestion
    handleStakingLimit(blockEvent, findings),
```

File: ethereum-steth/src/abi/ENS.json
Comment (dgusakov) [inline ethereum-steth/src/abi/ENS.json:1]: Pls remove unused ABIs from the new agents

**Review by dgusakov — CHANGES_REQUESTED** [review]

File: ethereum-steth/e2e/agent-steth-ops.e2e-spec.ts [hunk 5]
```diff
@@ -0,0 +1,213 @@
...
+    async () => {
+      let findings = await runBlock(17241600);
+      findings.sort();
+      expect(findings).toMatchSnapshot();
+    },
+    TEST_TIMEOUT,
+  );
+
+  // todo: disabled due to changes in the event signatures.
+  //  uncomment or remove after V2
```
[the remark this answers is not in the document]
Comment (vgorkavenko) [reply ethereum-steth/e2e/agent-steth-ops.e2e-spec.ts]: removed

File: ethereum-steth/package.json [hunk 6]
```diff
@@ -0,0 +1,40 @@
+{
+  "name": "lido-operations-forta-bot",
```
[the remark this answers is not in the document]
Comment (vgorkavenko) [reply ethereum-steth/package.json]: fixed

File: ethereum-governance/package.json
Comment (vgorkavenko) [reply ethereum-governance/package.json:1]: fixed

File: ethereum-steth/src/subagents/steth-ops/agent-steth-ops.ts [hunk 7]
```diff
@@ -26,53 +23,40 @@ import {
 } from "../../common/utils";
 import type * as Constants from "./constants";
 
-export const name = "DaoOps";
+export const name = "stETHops";
```
[the remark this answers is not in the document]
Comment (vgorkavenko) [reply ethereum-steth/src/subagents/steth-ops/agent-steth-ops.ts]: done

File: ethereum-steth/src/subagents/steth-ops/agent-steth-ops.ts (hunk 4 above)
Comment (vgorkavenko) [reply ethereum-steth/src/subagents/steth-ops/agent-steth-ops.ts]: removed

File: ethereum-validators-set/package.json [hunk 8]
```diff
@@ -0,0 +1,40 @@
+{
+  "name": "lido-operations-forta-bot",
```
[the remark this answers is not in the document]
Comment (vgorkavenko) [reply ethereum-validators-set/package.json]: fixed

File: ethereum-steth/src/abi/ENS.json
Comment (vgorkavenko) [reply ethereum-steth/src/abi/ENS.json:1]: done

**Review by dgusakov — APPROVED** [review]

**Pull request merged**